GeorgianMemenator/Scenes/MainView/MainViewModel.swift

```diff
@@ -18,6 +18,7 @@ class MainViewModel: ObservableObject {
         MemeCategory(title: "Cat status codes", icon: Image("cat")),
         MemeCategory(title: "Girchi", icon: Image("cone")),
         MemeCategory(title: "References", icon: Image("hand")),
+        MemeCategory(title: "Cat tags list", icon: Image("cats"))
```

Comment:
Please download the cat file from here https://www.flaticon.com/search?word=cat

Download .svg file from there and add it to Assets and use that icon here.